etc/skate-style/project.osm2pgsql.mml

```diff
@@ -475,7 +475,7 @@
         "key_field": "", 
         "project": "foss4g-2011", 
         "srs": "+proj=merc +a=6378137 +b=6378137 +lat_ts=0.0 +lon_0=0.0 +x_0=0.0 +y_0=0.0 +k=1.0 +units=m +nadgrids=@null +wktext +no_defs +over", 
-        "table": "( SELECT amenity AS type, name, way_area AS area,\n    ST_PointOnSurface(way) AS way\n  FROM planet_osm_polygon\n  WHERE name IS NOT NULL\n    AND way && !bbox!\n    AND ST_IsValid(way)\n UNION ALL\n\n  SELECT amenity AS type, name, 0 AS area, way\n FROM planet_osm_point\n  WHERE name IS NOT NULL\n ORDER BY area DESC\n) AS data", 
+        "table": "( SELECT osm_id, amenity AS type, name, way_area AS area,\n    ST_PointOnSurface(way) AS way\n  FROM planet_osm_polygon\n  WHERE name IS NOT NULL\n    AND way && !bbox!\n    AND ST_IsValid(way)\n UNION ALL\n\n  SELECT osm_id, amenity AS type, name, 0 AS area, way\n FROM planet_osm_point\n  WHERE name IS NOT NULL\n ORDER BY area DESC, osm_id\n) AS data", 
```

Comment:
note(non-blocking): can `osm_id` be left out of the top level query? I don't remember, but I thought you could `ORDER BY` without needing to `SELECT` it as long as it's present in the `FROM` but maybe I'm missing the full picture

Reply:
Unfortunately it seems like it is required - I tried removing, and got
```
An error occurred while loading the map layer 'default': Postgis Plugin: ERROR:  column "osm_id" does not exist
LINE 12:  ORDER BY area DESC, osm_id
```